pkg/apis/extensions/v1beta1/register.go

```diff
@@ -60,6 +58,7 @@ func addKnownTypes() {
 		&ListOptions{},
 		&ConfigMap{},
 		&ConfigMapList{},
+		&v1.DeleteOptions{},
```

Comment:
That's what this code is doing.  The internal version (the go type) is
irrelevant - it merely defines a schema.